### ISSUE TITLE
error loading connector widget

### ISSUE THREAD

**Issue by edublancas** [ploomber/jupysql-plugin#41]
starting from a new environment:

```sh

conda create --name tmp python=3.10 -y
conda activate tmp
pip install "jupyterlab<4" jupysql-plugin jupysql
jupyter lab
```

then in a notebook, run:

```python
from jupysql_plugin import __version__
__version__ # 0.1.5

from jupysql_plugin.widgets import ConnectorWidget
ConnectorWidget()
```

yields:

```
[Open Browser Console for more detailed log - Double click to close this message]
Failed to load model class 'ConnectorModel' from module 'jupysql-plugin'
loadClass@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/134.402424ef4079078b2e0e.js?v=402424ef4079078b2e0e:1:74976
loadModelClass@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.1a6d6a3a0542a41bec5a.js?v=1a6d6a3a0542a41bec5a:1:10729
_make_model@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.1a6d6a3a0542a41bec5a.js?v=1a6d6a3a0542a41bec5a:1:7517
new_model@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.1a6d6a3a0542a41bec5a.js?v=1a6d6a3a0542a41bec5a:1:5137
handle_comm_open@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/150.1a6d6a3a0542a41bec5a.js?v=1a6d6a3a0542a41bec5a:1:3894
134/v/this._handleCommOpen@http://localhost:8888/lab/extensions/@jupyter-widgets/jupyterlab-manager/static/134.402424ef4079078b2e0e.js?v=402424ef4079078b2e0e:1:73392
_handleCommOpen@http://localhost:8888/static/lab/jlab_core.87737d4fe404e2a19e9a.js?v=87737d4fe404e2a19e9a:2:1040012
```

@bbeat2782: you're right, there's a problem with this. let's pause this work until we fix it. I'll assign another issue shortly

**Comment by yafimvo** [comment]
@edublancas 
I was able to reproduce this bug.

Quick update:

I'm getting the same error for our examples widgets

i.e.
```
from jupysql_plugin._widgets import TableWidget
```

When I'm running it in [dev mode](https://github.com/ploomber/jupysql-plugin#development-install) it works as expected

**Comment by edublancas** [comment]
thanks for the update. my guess is that we introduced this error when we pushed some update. based on the error, maybe we're not including certain files in the package?

we build the release artifact with `python -m build`, please verify if anything looks odd when you generate it: https://github.com/ploomber/jupysql-plugin/blob/main/RELEASE.md

**Comment by yafimvo** [comment]
It looks like the build works ok.

First, I ran `python -m build` and it generated `jupysql_plugin-0.1.6-py3-none-any.whl` in `dist/`

Then I ran

```
conda create --name tmp python=3.10 -y
conda activate tmp
pip install "jupyterlab<4" jupysql jupysql_plugin-0.1.6-py3-none-any.whl
```

I was able to run ConnectorWidget successfully

<img width="696" alt="image" src="https://github.com/ploomber/jupysql-plugin/assets/103026689/09c42f40-7b8d-4363-bf92-93a00cab1b40">

**Comment by edublancas** [comment]
I found the problem, all the PyPI uploads have wrong metadata. the Python version doesn't match the JS package version, they all point to a lower version:

```
curl -O https://files.pythonhosted.org/packages/6e/f9/c4ce3523e233f545af2e7f07e324d59a6ad092dc7154ff0cc17bd8e9d1b4/jupysql_plugin-0.1.4-py3-none-any.whl
unzip -p jupysql_plugin-0.1.4-py3-none-any.whl "jupysql_plugin-0.1.4.data/data/share/jupyter/labextensions/jupysql-plugin/package.json" | head -n10
# says 0.1.3

curl -O https://files.pythonhosted.org/packages/1b/68/135fdeb78bf3195e70d4bfdad0348c211437578e9c81466dcbd3ad5acb54/jupysql_plugin-0.1.5-py3-none-any.whl
unzip -p jupysql_plugin-0.1.5-py3-none-any.whl "jupysql_plugin-0.1.5.data/data/share/jupyter/labextensions/jupysql-plugin/package.json" | head -n10
# says 0.1.4

curl -O https://files.pythonhosted.org/packages/c0/20/af7dd5621981f7d593ae8916c684ce17838700fa89ad363e8574165756df/jupysql_plugin-0.1.6-py3-none-any.whl
unzip -p jupysql_plugin-0.1.6-py3-none-any.whl "jupysql_plugin-0.1.6.data/data/share/jupyter/labextensions/jupysql-plugin/package.json" | head -n10
# says 0.1.5
```

I realized that `python -m build` does not perform from scratch, but uses whatever existing files there are. in my case, I had a `jupysql_plugin/labextension/package.json` file that had the wrong version and was used whenever I executed `python -m build`. I deleted the labextension/ directory, generated a new file, released it (0.1.7) and now it's working.

I'm still trying to fully understand the whole build process but for now, I'll delete the labextension/ directory before making any new release.

@yafimvo: can you verify that the 0.1.7 version uploaded to PyPI works?

I tested it and it works, although there is a react incompatibility (although this doesn't affect the widgets):

```
(tmp3) eduardo@macbookair dev/jupysql-plugin (main) » pip freeze | grep jupy
jupysql==0.7.9
jupysql-plugin==0.1.7
jupyter-events==0.6.3
jupyter-ydoc==0.2.4
jupyter_client==8.3.0
jupyter_core==5.3.1
jupyter_server==2.7.0
jupyter_server_fileid==0.9.0
jupyter_server_terminals==0.4.4
jupyter_server_ydoc==0.8.0
jupyterlab==3.6.5
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.8
jupyterlab_server==2.23.0
(tmp3) eduardo@macbookair dev/jupysql-plugin (main) » jupyter labextension list
JupyterLab v3.6.5
/Users/eduardo/miniconda3/envs/tmp3/share/jupyter/labextensions
        jupysql-plugin v0.1.7 enabled  X (python, jupysql-plugin)
        jupyterlab_pygments v0.2.2 enabled OK (python, jupyterlab_pygments)
        @jupyter-widgets/jupyterlab-manager v5.0.8 enabled OK (python, jupyterlab_widgets)


"jupysql-plugin@0.1.7" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLabExtension        Package
>=17.0.1 <18.0.0>=18.2.0 <19.0.0 react

```

I think this react incompatibility was introduced when we added the "deploy notebook" button. perhaps it's related to the other problem we encountered @tonykploomber ?

**Comment by yafimvo** [comment]
@edublancas I can confirm that it works from my side
